src/lib.rs

```diff
@@ -46,10 +43,18 @@ fn is_promise_success() -> bool {
 
 #[near_bindgen]
 impl LinkDrop {
+    /// Initializes the contract with an empty map for the accounts
+    #[init]
+    pub fn new() -> Self {
+        Self { 
+            accounts: UnorderedMap::new(b"a") 
```

Comment:
Nothing needs to change, but noting that this isn't the same prefix of the old `Map` default, but it shouldn't matter because it will load the old prefix if already initialized. The old prefix was `[0,0,0,0,0,0,0,0]`